mmdet3d/apis/inference.py

```diff
@@ -66,6 +66,7 @@ def init_model(config, checkpoint=None, device='cuda:0'):
     model.cfg = config  # save the config in the model for convenience
     model.to(device)
     model.eval()
+    model.init_weights()
```

Comment:
We do not need to initialize the model because it loads the checkpoint. If we initialize the model the checkpoint will have no effect.